warehouse/ingest-core/src/test/java/datawave/ingest/mapreduce/EventMapperTest.java

```diff
@@ -199,6 +202,115 @@ public void shouldNotWriteRawFile() throws IOException, InterruptedException {
         assertEquals(4, written.size());
     }
 
+    public static class BaseEventPredicate implements RawRecordPredicate {
```

Comment:
Consider breaking this into it's own class so there's an "AlwaysTrue" default implementation available, similar to how we construct a TimeFilter.

If these objects are going to be stored in a set then any implementation should override equals and hashcode.

Reply:
hmm, not a bad idea.  will do

Reply:
Since we pass in the classes by class name, the same mechanism as done in TimeFilter will not work exactly.  Making the base implementation abstract however and creating an explicit "AlwaysProcessPredicate" class to make that more obvious.